playground/Makefile

```
@@ -1,11 +1,14 @@
+COMPILER=5.0.0
+SWITCH=5.0.0
```

```suggestion
SWITCH=playground
```
I believe the compiler version is irrelevant there.